docs/src/app/components/CodeExample/index.js

```diff
@@ -40,9 +41,16 @@ class CodeExample extends React.Component {
       },
     };
 
+    const docs = parse(code);
```

Comment:
Yeah, that feel much better like this.
I think that we should follow this pattern everywhere (having the description above the component).
cc @mbrookes